providers/aws/services/vpc/vpc_endpoint_connections_trust_boundaries/vpc_endpoint_connections_trust_boundaries.metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Provider": "aws",
+  "CheckID": "vpc_endpoint_connections_trust_boundaries",
+  "CheckTitle": "Find trust boundaries in VPC endpoint connections.",
+  "CheckType": ["Infrastructure Security"],
+  "ServiceName": "vpc",
+  "SubServiceName": "endpoint",
+  "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
+  "Severity": "medium",
+  "ResourceType": "AwsEc2Vpc",
+  "Description": "Find trust boundaries in VPC endpoint connections.",
+  "Risk": "Account VPC could be linked to other accounts.",
+  "RelatedUrl": "",
+  "Remediation": {
+    "Code": {
+      "CLI": "",
+      "NativeIaC": "",
+      "Other": "https://docs.bridgecrew.io/docs/networking_9#aws-vpc-endpoints-are-exposed",
```

Comment:
I think in other we have to include the Ansible one here from the Indian company.

Reply:
https://www.cloudmatos.com/

Reply:
There is no such remediation for this check.